core/src/utils/forms/form-controller.ts

```diff
@@ -1,11 +1,13 @@
+type HTMLLegacyFormControlElement = HTMLElement & { label?: string };
```

Comment:
Will this work for ion-toggle? (it has a slot for the label instead of a prop)

Reply:
The `HTMLIonToggleElement` will satisfy the type constraint, since `label` is optional. This basically allows any `HTMLElement`, but solves for the custom property we are checking for in the legacy controller implementation (`label`).

Reply:
Sounds good!